tests/integration/v2/example/example_test.go

```
@@ -0,0 +1,93 @@
+	"cosmossdk.io/log"
+	mintkeeper "cosmossdk.io/x/mint/keeper"
+	minttypes "cosmossdk.io/x/mint/types"
+
+	"github.com/cosmos/cosmos-sdk/tests/integration/v2"
+	"github.com/cosmos/cosmos-sdk/testutil/configurator"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// Example shows how to use the integration test framework to test the integration of SDK modules.
+// Panics are used in this example, but in a real test case, you should use the testing.T object and assertions.
+func Example(t *testing.T) {
```

golangci-lint (1.62.2)

tests: Example should be niladic

(govet)